prime-router/metadata/schemas/covid-19.schema

```diff
@@ -673,8 +673,8 @@ elements:
 
   - name: processing_mode_code
     type: CODE
+    cardinality: ZERO_OR_ONE
     valueSet: hl70103
-    default: P
     hl7Field: MSH-11-1
```

Comment:
I see no reason to set this to 'P' - let's just wait until SImpleReport sends us valid values here.  When that happens, we'll need to change our jurisdictionFilters

Reply:
Agree